tests/functional/firefox/test_all.py

```diff
@@ -0,0 +1,26 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.all import FirefoxAllPage
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_search_language(base_url, selenium):
+    page = FirefoxAllPage(base_url, selenium).open()
+    page.type_search('english')
+    page.click_search()
```

Comment:
The click could still be needed in some browsers (which don't support the `input` event). Good suggestion on the abstraction though :+1: